src/main/java/frc/robot/Constants.java

```diff
@@ -103,6 +103,24 @@ public static final class ModuleConstants {
         public static final int DRIVE_CURRENT_LIMIT = 65;
         public static final int ANGLE_CURRENT_LIMIT = 25;
     }
+    public static final class IntakeConstants{
+        public static final Gains PIDGains = new Gains(0, 0, 0);
+        public static final Gains FFangleGains = new Gains(0, 0, 0);
+
+        public enum INTAKE_ANGLE {
+            GROUND(0),
+            AMP(0),
+            HUMAN_PLAER(0),
+            TO_SHOOTER(0),
+            IDLE(0);
```

Comment:
Explain to me how can I know an angle before we even have a robot מר בחור

Reply:
Bruu you dont have the robot but you know what it looks like and how it will be built..

Assuming 0 degrees is when the intake is parallel to the ground and inside the robot, keep going from there..
HM will be about 45 degrees..
Ground will be about 190 abd stuff like that

Reply:
I think it will be just better testing it and writing it down when we have the robot, theres no need for it now

Reply:
If you think so